cli_reference/odo-important-update.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/developer_cli_odo/odo-important-update.adoc
+
+:_content-type: CONCEPT
+[id="odo-important_update_{context}"]
+include::_attributes/common-attributes.adoc[]
+= Important update on `{odo-title}`
+:context: odo-important-update
+
+toc::[]
+
+Red Hat no longer provide documentation support for `{odo-title}`.
+
+[IMPORTANT]
+====
+This is an upstream community link:https://odo.dev[document] and it is not supported by Red Hat.
```

Comment:
Odo is not fully supported as most other Red Hat Products, but there is limited support by Red Hat.

Odo is supported under Cooperative Community Support 
(https://access.redhat.com/solutions/5893251).

It might be better to mention this rather than say it is not supported.